### DISCUSSION TITLE
PBSendSurvey : Permitir que los formularios se envien con estado "enviado"

### DISCUSSION THREAD

**Discussion by gonzalo-d** [Cotalker/documentation#710] [3. Ideas & Feature Requests]
Actualmente, PBSendSurvey permite enviar formularios de forma simple, e incluso prellenado con cierta data:

![image](https://user-images.githubusercontent.com/29810844/154740612-ab6a2beb-33ea-4d8d-9bba-2503780d75bc.png)

Sin embargo, un pequeño dolor que he tenido y que sería increible que se pudiera, es enviar estos formularios a traves de este bot, con estado "enviado". Actualmente lo más cercano que se puede hacer es prellenar todos los campos y luego enviarlos con un click. Esto si bien ahorra 1 segundo, permitiría una automatización más completa al enviar un formulario, agregando los datos directamente a la BD.

**Comment by acedward** [comment]
Agregar opción (https://doc.cotalker.com/docs/documentation/automation/bots/pbsendsurvey-3.0.0/)
`sendSaved: boolean`
